README.md

```diff
@@ -0,0 +1,89 @@
+<div align="center">
+  <br/>
+  <a href="https://github.com/zaproxy/zap-api-nodejs" title="zaproxy">
+    <img width=186px src="assets/images/ZapNodeApi.png" alt="zap-api-nodejs logo">
+  </a>
+  <br/>
+<br/>
+<h2>OWASP ZAP Node API</h2>
+<br/><br/>
+
+<a href="https://www.apache.org/licenses/LICENSE-2.0.html" title="License">
+  <img src="https://img.shields.io/badge/license-Apache%202-4EB1BA.svg" alt="License"/>
+</a>
+
+<br/><br/>
+</div>
+
+The NodeJS implementation to access the [OWASP ZAP API](https://github.com/zaproxy/zaproxy/wiki/ApiDetails). For more information
+about OWASP ZAP consult the (main) [OWASP ZAP project](https://github.com/zaproxy/zaproxy/).
+
+## Install
+
+```
+npm install zaproxy
+```
+
+## Usage
+
+By default Zap [requires an API key](https://github.com/zaproxy/zaproxy/wiki/FAQapikey) to be sent with every request. This is done automatically providing you supply the same API key when you instantiate the `ZapClient` that you use to run Zap with. All following API requests will use this same API key.  
```

Comment:
We should use `ZAP` instead of `Zap`, for consistency.

Reply:
That sould be it.